### ISSUE TITLE
AGPLv3

### ISSUE THREAD

**Issue by julialongtin** [kliment/explicitcad#1]
Newer versions of ImplicitCAD are AGPLv3. It would be nice if this tool was also the same license.

**Comment by hannesweisbach** [comment]
Sorry to hijack your issue @julialongtin, but do you know if the difference in licenses would prevent a package containing both ExplicitCAD and ImplicitCAD being built/distributed? Same goes for Qt?

**Comment by julialongtin** [comment]
nope, it wouldn't. afaik, implicitcad is used from it's external interface by explicitcad.

speaking of hijacking issues, you really should start using the implicitsnap binary instead. that way you have an implicitcad daemon to talk to, and can stop starting and stopping implicitcad.

and thanks for taking this on. :)

**Comment by hannesweisbach** [comment]
> nope, it wouldn't. afaik, implicitcad is used from it's external interface by explicitcad.

It is. Currently ExplicitCAD just calls ImplicitCAD as seperate process.


> speaking of hijacking issues, you really should start using the implicitsnap binary instead. that way you have an implicitcad daemon to talk to, and can stop starting and stopping implicitcad.

Hmm, that sounds nice. But before making any serious code-changes I want to get CI up and running :)

> and thanks for taking this on. :)

👍
